app/javascript/components/applications/application_form/formOptions.js

```diff
@@ -1,11 +1,12 @@
-import { isEmpty, map } from 'lodash'
+const isNullOrEmptyString = (value) => value === undefined || value === null || value === ''
 
 const labelize = (options, attrs = {}, noPlaceholder = false) => {
-  if (isEmpty(options)) return []
+  if (options.length === 0) return []
 
   let emptyInitialOptionPresent =
-    isEmpty(options[0]) ||
-    (options[0].hasOwnProperty('value') && isEmpty(options[0].value)) || noPlaceholder
```

Comment:
`isEmpty(options[0].value)` here is a bug. [Lodash isEmpty](https://lodash.com/docs/4.17.15#isEmpty) isn't meant to be called on strings and null values don't work as expected.